client/my-sites/plans-features-main/index.jsx

```diff
@@ -261,8 +273,11 @@ export class PlansFeaturesMain extends Component {
 	}
 
 	renderToggle() {
-		const { displayJetpackPlans, withWPPlanTabs } = this.props;
+		const { displayJetpackPlans, withWPPlanTabs, countryCode } = this.props;
 		if ( displayJetpackPlans ) {
+			if ( abtest( 'onlyJetpackMonthly', countryCode ) !== 'original' ) {
```

Comment:
Perhaps we could avoid a separate nested `if` if we invert the condition and move it to the top `if`?

Reply:
but that would add an indirection instead of a simple 'local' if check. I would rather avoid calling external functions (`abtest()`) than avoid an extra if ;)

(Also, there's another line inside that `if` that's not affected by the test, so it would have to be expanded into two different if branches to replicate the logic)